app/models/user.rb

```diff
@@ -22,8 +22,6 @@ class User < ApplicationRecord
   has_many :appropriate_body_profiles, dependent: :destroy
   has_many :appropriate_bodies, through: :appropriate_body_profiles
 
-  has_many :npq_application_exports
```

Comment:
Any reason to remove this? It's used as the inverse on NPQApplications::Export's user relation.

Reply:
This is duplicated below